client/my-sites/site-settings/disconnect-site/style.scss

```diff
@@ -32,3 +30,9 @@
 			text-align: center;
 		}
 }
+
+.disconnect-site__back-button {
```

Comment:
By using percentages here, the button will “float” around depending on the screen's resolution. Instead, we can just use the same styles as `.reader-full-post__back-container` (ref: https://github.com/Automattic/wp-calypso/blob/64d9a90c27965d2af288d977b28318e8f95ace14/client/blocks/reader-full-post/style.scss#L37-L60), which in this scenario would be:

```
.disconnect-site__back-button {
	margin: 0;
	position: fixed;
		top: 47px;
		left: 0;
	z-index: z-index( '.masterbar', '.reader-back'	);

	.button.is-compact.is-borderless {
		padding: 18px 40px 18px 20px;

		@include breakpoint( "<660px" ) {
			padding: 20px 8px 16px;
		}
	}

	@include breakpoint( "<660px" ) {
		top: 0;
		left: 0;
		right: 0;
		height: 46px;
		background: $white;
		border-bottom: 1px solid $gray-light;
	}
}
```

Reply:
Makes sense, thanks @keoshi.

Reply:
I also moved the button specific styling to the component's style file. IMHO the idea would be to define button's container styling when using it (e.g. its positioning ), and not worry about the looks of it.